sdk/communication/Azure.Communication.Chat/tests/SessionRecords/ChatClientsLiveTests/ReadReceiptGSAsyncAsync.json

```diff
@@ -240,82 +261,91 @@
       "ResponseHeaders": {
         "api-supported-versions": "2020-09-21-preview2, 2020-11-01-preview3, 2021-01-27-preview4",
         "Content-Type": "application/json; charset=utf-8",
-        "Date": "Fri, 05 Feb 2021 04:50:19 GMT",
-        "Location": "https://chat-sdktester-e2e.communication.azure.com/chat/threads/19%3AMGFVz7lbRHfSeRIAyD9rXZqx5i4TAGdAemi1DcEuvRE1@thread.v2/messages/1612500620132",
-        "MS-CV": "w5Y7nw8l0kGtCExOtkDFsg.0",
+        "Date": "Fri, 12 Feb 2021 03:08:49 GMT",
+        "Location": "https://smstestapp.communication.azure.com/chat/threads/19%3AKKlEhjwvup9_n9n1Qlq5itO5y-ZitOvWWcX11iMBlKA1@thread.v2/messages/1613099329845",
```

Comment:
I think resource names are fine to be left unsansitized as they get destroyed, but for consistency sake should this be sanitized too, or unsanitize the rest?

Reply:
Yeah the Location needs sanitization. Resource name I think should be sanitized

Reply:
I pushed a new update that sanitize the location header as well

Reply:
FYI @RezaJooyandeh they are not required to be sanitized, along with IP adress, or IDs, we ran that few times by the azure folks long ago.

Reply:
You are right, looking around in other libraries as well it seems that we do not sanitize the urls anymore. I will remove it